src/translations/fr.json

```
@@ -1602,6 +1602,10 @@
     "searchVoltageLevelInNad": "Rechercher un poste",
     "noResultsFound": "Aucun résultat trouvé",
     "hide": "Masquer",
+    "hideAll": "Tout masquer",
+    "showAll": "Tout afficher",
+    "grid": "Disposition en grille",
+    "cascade": "Disposition en cascade",
```

```suggestion
    "gridLayout": "Disposition en grille",
    "cascadeLayout": "Disposition en cascade",
```

to prevent future collisions